exercises/practice/state-of-tic-tac-toe/.meta/src/reference/java/GameStates.java

```diff
@@ -0,0 +1,5 @@
+enum GameStates {
```

Comment:
Nitpicking perhaps, but I believe the convention in Java is to use singular names for enums, as you're only selecting one value: `GameState.WIN`. It also reads more natural.

Reply:
Done!